chromadb/test/db/test_system.py

```diff
@@ -139,23 +143,26 @@ def test_create_get_delete_collections(sysdb: SysDB) -> None:
 
     for collection in sample_collections:
         sysdb.create_collection(
-            id=collection["id"],
-            name=collection["name"],
+            id=collection.id,
+            name=collection.name,
+            configuration=collection.get_configuration(),
             metadata=collection["metadata"],
             dimension=collection["dimension"],
         )
         collection["database"] = DEFAULT_DATABASE
         collection["tenant"] = DEFAULT_TENANT
 
     results = sysdb.get_collections()
-    results = sorted(results, key=lambda c: c["name"])
+    results = sorted(results, key=lambda c: c.name)
 
-    assert sorted(results, key=lambda c: c["name"]) == sample_collections
+    assert sorted(results, key=lambda c: c.name) == sample_collections
```

Comment:
I think these tests rely on the __eq__ of CollectionCommon.py - which we need to update for these tests to actually check the configuration